src/main/java/frc/robot/subsystems/ArmControlSubsystem.java

```diff
@@ -0,0 +1,306 @@
+    chooser.addOption("Coast", ArmMotorMode.COAST); 
+    chooser.setDefaultOption("Brake", ArmMotorMode.BRAKE);
+
+    SmartDashboard.putData("ArmMotorMode", chooser);
+    
+  }
+
+  public void setConfig(){
+    _rightPivotController.configFactoryDefault();
+    _leftPivotController.configFactoryDefault();
+
+    _rightPivotController.follow(_leftPivotController);
+
+    _rightPivotController.setInverted(TalonFXInvertType.OpposeMaster);
+    _leftPivotController.setInverted(ArmConstants.kLeftPivotInverted);
+
+    this.updateModes();
+
+    _rightPivotController.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
+    _leftPivotController.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
+
+    _leftPivotController.setSelectedSensorPosition(0);
+    _rightPivotController.setSelectedSensorPosition(0);
+    
+    extensionController.setInverted(false);
+
+    extensionEncoder.setPosition(0);
+
+    extensionController.burnFlash();
+    
+  }
+
+  @Override
+  public void periodic() { 
+    if(chooser.getSelected() == ArmMotorMode.BRAKE && _isInitialized){
+      pivotPeriodic();
+      extensionPeriodic();
+    }else if(chooser.getSelected() == ArmMotorMode.COAST){
+      _desiredPivotRotation = _currentPivotRotation;
+      _desiredExtensionDistance = _currentExtensionDistance;
+    }
+
+    if(!_isInitialized && -_absPivEncoder.getAbsolutePosition() * ArmConstants.kPivotAbsEncToRotation != 0 && _absPivEncoder.isConnected()){
+      resetEncoders();   
+      _isInitialized = true;
+    }
+    
+    updateModes();
+  
+
+    // Getting Current And Desired Distances
+    SmartDashboard.putNumber("CurrentExtension", this.getCurrentExtensionIn());
+    SmartDashboard.putNumber("CurrentPivotDeg", Units.radiansToDegrees(_currentPivotRotation));
+
+    if (CompConstants.kDebugMode) {
+      //Encoder Positions
+      SmartDashboard.putNumber("PivotPos", getCurrentPivotRotation(false));
+      SmartDashboard.putNumber("InitialAbsPivot", this._pivotRelEncoderOffsetRot);
+
+      SmartDashboard.putNumber("extensionSensorOutput", getCurrentExtensionIn());
+      SmartDashboard.putNumber("extensionEncoderPos", extensionEncoder.getPosition()*ArmConstants.kExtensionRotationToInches);
+    
+      //Abs Encoder Debugging
+      SmartDashboard.putNumber("pivotfreq", _absPivEncoder.getFrequency());
+      SmartDashboard.putBoolean("IsInititilized", _isInitialized);
+      SmartDashboard.putBoolean("absreconnectioned", _absPivEncoder.isConnected());
+      SmartDashboard.putNumber("AbsPivot", -_absPivEncoder.getAbsolutePosition() * ArmConstants.kPivotAbsEncToRotation);
+
+      //Setpoint Debugging
+      SmartDashboard.putBoolean("AtAnglePoint", this.atAngleSetpoint());
+      SmartDashboard.putBoolean("AtExtensionPoint", this.atTelescopeSetpoint());
+      SmartDashboard.putNumber("DesiredPivotDeg", Units.radiansToDegrees(_desiredPivotRotation));
+      SmartDashboard.putNumber("DesiredExtension", _desiredExtensionDistance);
+
```

Comment:
Please go through the PR and remove all extraneous empty lines, and remove all extraneous uses of this.

Reply:
And eliminate use of magic numbers. That is, most uses of constant numeric values should be upgraded to use named constants. This tells the reader a lot more information than merely seeing 0.15